heron/tools/cli/src/python/response.py

```diff
@@ -68,33 +70,55 @@ def add_context(self, err_context, succ_context=None):
     self.err_context = err_context
     self.succ_context = succ_context
 
+def do_log(log, msg):
```

Comment:
do_log and log_context seem like they are intended to be private. If that's the case please move them to the bottom and prepend `_` to the method name to make that clear.